CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### `items.xml` enhancements
 - Improved folding, show '!' for mandatory properties and relations [#1009](https://github.com/epam/sap-commerce-intellij-idea-plugin/pull/1009)
```

Comment:
let's add 2nd PR to the same line, as 
`#1009 | #1010`

Reply:
done